softvis3d-frontend/package.json

```diff
@@ -64,6 +64,8 @@
     "css-loader": "^0.26.1",
     "enzyme": "^2.7.0",
     "extract-text-webpack-plugin": "^1.0.1",
+    "jsdom": "9.10.0",
```

Comment:
Great, didn't see the difference in the test case ;-) If mount() is better than shallow(), shoudn't we update all existing test cases from swallow to mount (i could do that) to keep it all the same.